README.md

```diff
@@ -66,4 +66,4 @@ in our repository in the `COPYING` file.
 
 ## Contributing
 
-As an open source project, Klaytn is always welcoming your contribution. Please read our [CONTTIBUTING.md](./CONTRIBUTING.md) for a walk-through of the contribution process.
+As an open source project, Klaytn is always welcoming your contribution. Please read our [CONTRIBUTING.md](./CONTRIBUTING.md) for a walk-through of the contribution process.
```

Comment:
```suggestion
As an open source project, Klaytn always welcomes your contribution. Please read our [CONTRIBUTING.md](./CONTRIBUTING.md) for a walk-through of the contribution process.
```
Very good!
Could you please make one more change?
As fas as I know, be always ~ing is a negative expression.

Reply:
Sure :) I've updated the PR. Please take a look again.